### PR TITLE
travis test all file

### PR DESCRIPTION
**Pull request by ductm54**

### CODE REVIEW

File: .travis/test-part.sh [hunk 1]
```diff
@@ -6,8 +6,8 @@ set -euxo pipefail
 readonly test_part=${TEST_PART:-}
 
 case "$test_part" in
-Sol6)
-    npx buidler test --no-compile --config buidlerConfigSol6.js
+Test)
```
Comment (ilanDoron) [inline .travis/test-part.sh:9]: in test file can maybe leave both options
so later we can configure as we want for yml file

Comment (ilanDoron) [reply .travis/test-part.sh:9]: actually seems more clear to call it All

and leave sol6 as well.

File: .travis.yml [hunk 2]
```diff
@@ -13,6 +13,8 @@ env:
 
 jobs:
   include:
+    - env:
+        - TEST_PART: All
```
Comment (ilanDoron) [inline .travis.yml:17]: when written like this it will test both All and Sol6?

File: buidler.config.js [hunk 3]
```diff
@@ -134,7 +134,8 @@ module.exports = {
             "0x28d1bfbbafe9d1d4f5a11c3c16ab6bf9084de48d99fbac4058bdfa3c80b2908f",
           balance: "100000000000000000000000000000000"
         }
-      ]
+      ],
+      allowUnlimitedContractSize: true
```
Comment (ilanDoron) [inline buidler.config.js:138]: why do we need this part?

File: buidlerConfigSol6.js [hunk 4]
```diff
@@ -135,7 +135,8 @@ module.exports = {
             "0x28d1bfbbafe9d1d4f5a11c3c16ab6bf9084de48d99fbac4058bdfa3c80b2908f",
           balance: "100000000000000000000000000000000"
         }
-      ]
+      ],
+      allowUnlimitedContractSize: true
```
Comment (ilanDoron) [inline buidlerConfigSol6.js:139]: and if we need it.
is it required both sol4 and sol6?

File: test/sol4/kyberProxyV1.js [hunk 5]
```diff
@@ -1571,19 +1571,16 @@ contract('KyberProxyV1', function(accounts) {
         });
 
         it("verify sell with malicious network2 reverts when using any min rate (0).", async function () {
-            let amountTwei = 1123;
+            let amountTwei = new BN(1000);
 
-            //set steal amount to 1 wei
-            let myFee = 1;
+            //set steal amount to 100 wei
+            let myFee = 100;
```
Comment (ilanDoron) [inline test/sol4/kyberProxyV1.js:1577]: 1 wei doesn't work anymore?

File: test/sol6/kyberNetwork.js [hunk 6]
```diff
@@ -1060,7 +1060,8 @@ contract('KyberNetwork', function(accounts) {
                 });
 
                 // 2 types of trade that specific which reserve will be choosen
```
Comment (ilanDoron) [inline test/sol6/kyberNetwork.js:1062]: if we are here.
can you fix the comment

seems it should have
specific --> specify

choosen --> chosen

File: buidler.config.js (hunk 3 above)
Comment (Anyhowclick) [reply buidler.config.js:138]: The malicious network contracts exceeds the max bytecode size. Without this, tests will fail cos the contracts can't be deployed (out of gas txns).

File: buidlerConfigSol6.js (hunk 4 above)
Comment (Anyhowclick) [reply buidlerConfigSol6.js:139]: Only sol6, cos it's only the malicious network contracts that exceed.

**Pull request merged**